apps/server/src/modules/learnroom/uc/rooms.uc.ts

```diff
@@ -0,0 +1,78 @@
+import { Injectable, NotFoundException, UnauthorizedException } from '@nestjs/common';
+import { EntityId, Course, Task, TaskWithStatusVo } from '@shared/domain';
+import { CourseRepo, TaskRepo } from '@shared/repo';
+
+// TODO: move this somewhere else
```

Comment:
still need to move this?

Reply:
yes, but into a class that doesnt exist yet. we will do that in a future sprint.